components/clearbutton/stories/clearbutton.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as ClearButton from './clearbutton.stories';
+
+<Meta of={ClearButton} title="Docs" />
+
+<Title of={ClearButton} />
+<Description of={ClearButton} />
+
+### Usage Notes
+
+Use the correct cross icon size that corresponds to the t-shirt size you require for the clear button.
+
+<table>
+  <thead>
+    <tr>
+      <th>**T-Shirt Size**</th>
+      <th>**Icon Size**</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>spectrum-ClearButton--sizeS</td>
+      <td>spectrum-css-icon-Cross75</td>
+    </tr>
+    <tr>
+      <td>spectrum-ClearButton--sizeM</td>
+      <td>spectrum-css-icon-Cross100</td>
+    </tr>
+    <tr>
+      <td>spectrum-ClearButton--sizeL</td>
+      <td>spectrum-css-icon-Cross200</td>
+    </tr>
+    <tr>
+      <td>spectrum-ClearButton--sizeXL</td>
+      <td>spectrum-css-icon-Cross300</td>
+    </tr>
+  </tbody>
+</table>
+
+### Standard
+
+This variant defaults to the `size: m` T-Shirt size.
```

Comment:
Because I didn't keep the t-shirt sizing stuff, I put a little blurb here about it. I think all of them default to the medium size, so maybe that should just go up near the top?

Reply:
You can remove this sentence since we're defaulting to medium for all the components.

Reply:
fixed here: https://github.com/adobe/spectrum-css/pull/2793/commits/2f3de46501e7bef35a93740d557063b3f3fe1af9